.github/workflows/master.yml

```diff
@@ -314,6 +314,8 @@ jobs:
         run: |
           export PATH="$CONDA_DIR/bin:$PATH"
           source activate test-environment
+          # Install libopenblas-dev for mxnet 1.8.0.post0
+          sudo apt-get install libopenblas-dev
           ./dev/run-python-flavor-tests.sh;
```

Comment:
Do we still need to run all of the tests in `tests/gluon`, `test/sklearn`, etc as part of `run-python-flavor-tests`? We have cross version coverage for these already, so it seems redundant.

Reply:
I think we can skip running them if we modify `dev/run-python-flavor-tests.sh`.

Reply:
Can we do that instead of adding a dependency on `libopenblas` for master workflows? This should also save us some CI time.

Reply:
I think we can, but this sounds like a follow-up.

Reply:
I think we can, but `sudo apt-get install libopenblas-dev` just takes 5 seconds (https://github.com/mlflow/mlflow/pull/4627/checks?check_run_id=3237179233#step:6:12).

Reply:
Right, but running a bunch of redundant tests takes longer. Can we do this as a follow-up? Feel free to file an internal ticket.

Reply:
Agree, I’ll file a follow up ticket.